data/templates/telegraf/telegraf.j2

```diff
@@ -45,7 +45,11 @@
 ### Prometheus ###
 [[outputs.prometheus_client]]
   ## Address to listen on
+{%     if prometheus_client.listen_address | is_ipv6 %}
+  listen = "[{{ prometheus_client.listen_address if prometheus_client.listen_address is vyos_defined else '' }}]:{{ prometheus_client.port }}"
```

Comment:
We have “bracketize_ipv6” function
For example 
https://github.com/vyos/vyos-1x/blob/627cfc6d6733998a721ac97f9f5bc44c4c2c6797/data/templates/snmp/etc.snmpd.conf.j2#L48